js/btcalpha.js

```diff
@@ -291,6 +299,113 @@ module.exports = class btcalpha extends Exchange {
         return this.parseTrades (trades, market, since, limit);
     }
 
+    async fetchDeposits (code = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const deposits = await this.privateGetDeposits (params);
+        //
+        //     [
+        //         {
+        //             "timestamp": 1485363039.18359,
+        //             "id": 317,
+        //             "currency": "BTC",
+        //             "amount": 530.00000000
+        //         }
+        //     ]
+        //
+        return this.parseTransactionsByType ('deposit', deposits, code, since, limit);
```

Comment:
Let's remove parseTransactionsByType, cause parseTransactions supports injections with the `params` argument:

```JavaScript
return this.parseTransactions (deposits, code, since, limit, { 'type': 'deposit' });

return this.parseTransactions (deposits, code, since, limit, { 'type': 'withdrawal' });
```